bezier_library/src/bezier_grinding_surfacing.cpp

```diff
@@ -320,6 +320,63 @@ std::string BezierGrindingSurfacing::generateTrajectory(EigenSTL::vector_Affine3
       applyLeanAngle(pose, axis_of_rotation_, lean_angle_);
   }
 
+  // The first pose of the trajectory is a point located above the first grinding pose.
+  // We use the last pose of the last extrication trajectory which is located above the
+  // first pose of the first grinding line. This point is stored as the first pose of the trajectory.
+  Eigen::Affine3d start_pose(extrication_trajectories.back().back());
+  // We keep the same orientation than the one of the first grinding pose of the first grinding line
+  start_pose.linear() << grinding_trajectories.front().front().linear();
+
+  // Generate Intermediate poses between the start pose of the trajectory and the first grinding line
+  EigenSTL::vector_Affine3d first_grinding_line_intermediate_poses;
+  Eigen::Vector3d end_pose(grinding_trajectories.front().front().translation());
+  generateIntermediatePoseOnLine(first_grinding_line_intermediate_poses, start_pose.translation(), end_pose, 2);
+  for (Eigen::Affine3d &pose : first_grinding_line_intermediate_poses)
+  {
+    // The intermediate poses have the same orientation than the first pose of the first grinding line
```

Comment:
I'm not sure that is the best option. I would rather interpolate the orientation between the last extrication pose and the first grinding pose.

However, this seems to work well as is so I'll merge and we'll improve the feature if needed.